iptables/table.go

```diff
@@ -205,6 +208,12 @@ type Table struct {
 	// to what we calculate from chainToContents.
 	chainToDataplaneHashes map[string][]string
 
+	// chainToFullRules contains the full rules, mapped from chain name to slices of rules in that chain.
+	chainToFullRules map[string][]string
```

Comment:
Worth commenting that this is partial information, limited to the chains that we expect to hook.